src/webgl/p5.RendererGL.js

```diff
@@ -1796,6 +1821,21 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
     fillShader.bindTextures();
   }
 
+  // getting called from _setFillUniforms
+  _setImageLightUniforms(shader){
+    //set uniform values
+    if( this.activeImageLight == null ){
+      console.log('activeImageLight prop is null');
+    }
+    shader.setUniform('uUseImageLight', this.activeImageLight != null );
+    // true
+    if (this.activeImageLight) {
+      // this.activeImageLight has image as a key
+      // look up the texture from the blurryTexture map
+      shader.setUniform('equiRectangularTextures', this.blurryTextures.get(this.activeImageLight));
```

Comment:
If we decide to make `getBlurryTexture` compute and store the blurry texture if it doesn't yet exist, instead of doing `blurryTextures.get()` here, we could do `this.getBlurryTexture(this.activeImageLight)`